ui/src/components/layout/navbar/dept-account-icon.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { useState } from 'react';
+
+import DynamicModal from '@/components/modal/dynamic-modal';
+import { faUser, faSchool } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip';
+import User from '@/lib/access/user';
+import Role from '@/lib/access/role';
+
+const DeptAccountIcon = ({ currentUser }: { currentUser: User }) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const openDepartmentalModal = () => {
+        setIsModalOpen(true);
+    };
+
+    const closeDepartmentalModal = () => {
+        setIsModalOpen(false);
+    };
+
+    return (
+        <>
+            {currentUser.roles.includes(Role.DEPARTMENTAL) && (
+                <div
+                    onClick={openDepartmentalModal}
+                    className="flex justify-center items-center rounded-full
+                            h-[45px] w-[45px] bg-seafoam mx-auto relative lg:ml-24"
+                >
+                    <TooltipProvider>
+                        <Tooltip>
+                            <TooltipTrigger>
+                                <FontAwesomeIcon aria-label="user" icon={faUser} width={14} height={16} />
+                                <div
+                                    className="bg-blue-background rounded-full flex justify-center
+                                    items-center h-[20px] w-[25px] absolute left-7 bottom-0"
```

Comment:
Try h-6 w-6 for the width and height, let me know if you think it looks better. The current icon looks off center.

Reply:
Using h-6 and w-6 will make the blue background into more a circular shape. I like this one more than how it looked before with it being more oval-shaped.
![image](https://github.com/user-attachments/assets/7005766d-c2da-41ca-9e2c-64b942702722)